monai/auto3dseg/data_analyzer.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import warnings
+from os import path
+from typing import Dict, List, Optional, Union
+
+import numpy as np
+import torch
+
+from monai import data
+from monai.apps.utils import get_logger
+from monai.auto3dseg.seg_summarizer import DataStatsKeys, SegSummarizer
+from monai.auto3dseg.utils import datafold_read
+from monai.bundle.config_parser import ConfigParser
+from monai.data.utils import no_collation
+from monai.transforms import (
+    Compose,
+    EnsureChannelFirstd,
+    EnsureTyped,
+    Lambdad,
+    LoadImaged,
+    Orientationd,
+    SqueezeDimd,
+    ToDeviced,
+)
+from monai.utils import min_version, optional_import
+from monai.utils.enums import ImageStatsKeys
+
+tqdm, has_tqdm = optional_import("tqdm", "4.47.0", min_version, "tqdm")
+logger = get_logger(module_name=__name__)
+
+__all__ = ["DataAnalyzer"]
+
+
+class DataAnalyzer:
```

Comment:
this should be in the apps folder because the transforms are hard-coded for a particular type of tasks, what do you think? @Nic-Ma @mingxin-zheng

Reply:
I agree.

Thanks.

Reply:
Sounds good

Reply:
I think we can put auto3dseg for now and discuss with Prerna later.

Reply:
Okay I am changing the folder structure now

Reply:
Fixed in bb02d7d05b907f1b4220d59140fabc7fb72097bc. Thanks!